### PR TITLE
UCS: new module to find vlan(s) based on different criteria

### PR DESCRIPTION
**Pull request by dx0xm**
##### SUMMARY
Created to check existing vlan(s) based on a particular criteria. Useful for doing pre checks and maintenance jobs

##### ISSUE TYPE
- New Module Pull Request

##### COMPONENT NAME
ucs_vlan_find

### CODE REVIEW

File: lib/ansible/modules/remote_management/ucs/ucs_vlan_find.py [hunk 1]
```diff
@@ -0,0 +1,122 @@
...
+    if module.params['vlanid']:
+        filtls.append('(id,"' + module.params['vlanid'] + '")')
+    else:
+        filtls.append('(name,"' + module.params['pattern'] + '")')
+
+    object_dict = ucs.login_handle.query_classid("fabricVlan", filter_str=' and '.join(filtls))
+
+    if object_dict is None:
+        module.fail_json(msg="Failed to query vlan objects")
+        return 0
```
Comment (Akasurde) [inline lib/ansible/modules/remote_management/ucs/ucs_vlan_find.py]: This return is useless.

**Review by Akasurde — COMMENTED** [review]
Minor nit-pick. Rest LGTM

File: lib/ansible/modules/remote_management/ucs/ucs_vlan_find.py (hunk 1 above)
Comment (dx0xm) [reply lib/ansible/modules/remote_management/ucs/ucs_vlan_find.py]: Thanks @Akasurde. Good point...fixed.

**Comment by dx0xm** [comment]
Hi @vvb , do you know who can help with this pull request?

Thanks.

**Review by dsoper2 — APPROVED** [review]
shipit

**Pull request merged**